src/Framework/MSBuildEventSource.cs

```diff
@@ -423,6 +423,18 @@ public void PacketReadSize(int size)
         {
             WriteEvent(55, size);
         }
+
+        [Event(56, Keywords = Keywords.All)]
+        public void PerformDependencyAnalysisStart()
```

Comment:
I think I'd prefer a name like `TargetUpToDateCheckStart`/Stop; as the comment above notes this "isn't really dependency analysis" so I'd love to avoid propagating the bad name. Thoughts?

Reply:
I slightly prefer it this way because 1) there is other work that happens when it does an UTD check, albeit not too much, and 2) consistency is good even if it's slightly wrong consistency. I don't feel at all strongly, though, so I can change it.